docs/user-guide.rst

```diff
@@ -207,6 +207,9 @@ The backends available are:
   You need to provide **Sourceforge name** if the name on RSS feed is different then the
   project name on Sourceforge.
 
+* **Sourceforge (git) for projects hosted on
```

Comment:
It will be good to describe the difference between this backend and the sourceforge one.

Reply:
Ok! Will do that.